client/data/account-keys/actions.js

```diff
@@ -111,3 +111,44 @@ export function* testAccountKeys( { live, publishable, secret } ) {
 
 	return error === null;
 }
+
+export function updateIsConfiguringWebhooks( isProcessing ) {
+	return {
+		type: ACTION_TYPES.SET_IS_CONFIGURING_WEBHOOKS,
+		isProcessing,
+	};
+}
+
+export function* configureWebhooks( { live, secret, callback } ) {
```

Comment:
This works great; I'm just curious. Is there any difference between using this as a generator function and using async/await?

I see we're using generators in other places too and this works well, so not asking to update it.

Reply:
Good question. I'm not 100% sure 😅 

I copied the precedence set by the other function in this file. TBH I wasn't really sure what the pros/cons were of using a generator function vs async but ChatGPT did say async is generally my preferred.

Reply:
I'm going to leave this as a generator function for now. We can change it to async in a separate change if we prefer that approach. I'd prefer to keep this PR moving. :)